mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -1,13 +1,19 @@
 CLASS net/minecraft/class_127 net/minecraft/entity/LivingEntity
 	FIELD field_1009 maxHealth I
 	FIELD field_1019 texture Ljava/lang/String;
+	FIELD field_1024 scoreAmount I
+	FIELD field_1029 sideways F
```

Comment:
I think `sideways` should be `sidewaysSpeed`
see https://maven.fabricmc.net/docs/yarn-1.21+build.6/net/minecraft/entity/LivingEntity.html#sidewaysSpeed